lib/gunit/SConscript

```diff
@@ -26,44 +26,51 @@ env.VariantDir(Dir('.').abspath + '/gtest', gtest_path + '/src')
 env.VariantDir(Dir('.').abspath + '/gmock', gmock_path + '/src')
 
 gmocklibname = 'gmock.lib'
-gtestlibname = 'gtest{d}.lib'.format(d='d' if env['TARGET_CONFIG'] =='debug' else  '')
+gtestlibname = 'gtest.lib'
```

Comment:
In windows or linux, the thrift version is generally fixed (even across various releases) in Contrail. 
But this is a very good suggestion . We can fix similar third party issues in some later fixes.